Assets/MixedRealityToolkit/_Core/Inspectors/Profiles/MixedRealityBoundaryVisualizationProfileInspector.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.﻿
+
+using Microsoft.MixedReality.Toolkit.Internal.Definitions.BoundarySystem;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Inspectors
+{
+    [CustomEditor(typeof(MixedRealityBoundaryVisualizationProfile))]
+    public class MixedRealityBoundaryVisualizationProfileInspector : MixedRealityBaseConfigurationProfileInspector
+    {
+        private SerializedProperty floorPlaneScale;
+        private SerializedProperty playAreaMaterial;
+        private SerializedProperty floorPlaneMaterial;
+        private SerializedProperty trackedAreaMaterial;
+
+        private readonly GUIContent scaleContent = new GUIContent("Scale");
+        private readonly GUIContent materialContent = new GUIContent("Material");
+
+        private void OnEnable()
+        {
+            if (!CheckMixedRealityManager(false))
+            {
+                return;
+            }
+
+            floorPlaneScale = serializedObject.FindProperty("floorPlaneScale");
+            playAreaMaterial = serializedObject.FindProperty("playAreaMaterial");
+            floorPlaneMaterial = serializedObject.FindProperty("floorPlaneMaterial");
+            trackedAreaMaterial = serializedObject.FindProperty("trackedAreaMaterial");
+        }
+
+        public override void OnInspectorGUI()
+        {
+            RenderMixedRealityToolkitLogo();
+            EditorGUILayout.LabelField("Boundary Visualization Options", EditorStyles.boldLabel);
+            EditorGUILayout.HelpBox("Boundary visualizations can help users stay oriented and comfortable in the experience.", MessageType.Info);
+            EditorGUILayout.Space();
+
```

Comment:
we need to also check if the MR Manger is valid here with

```
            if (!CheckMixedRealityManager())
            {
                return;
            }
```

Reply:
made this change... not sure why GitHub didn't notice.